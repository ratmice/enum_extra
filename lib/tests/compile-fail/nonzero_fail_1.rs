#[allow(unused_imports)]
use enum_extra::NonZeroRepr;
use strum::EnumMetadata;

#[derive(EnumMetadata, NonZeroRepr)]
#[repr(u8)]
enum ABC {
    C = 2,
    B = 1,
    A = 0,
}

fn main() {
}


