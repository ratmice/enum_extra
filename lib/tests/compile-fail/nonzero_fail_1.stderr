error: proc-macro derive panicked
 --> tests/compile-fail/nonzero_fail_1.rs:5:24
  |
5 | #[derive(EnumMetadata, NonZeroRepr)]
  |                        ^^^^^^^^^^^
  |
  = help: message: assertion failed: discriminant != 0
